src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -313,6 +311,8 @@ export class VSCodeNotebookController implements Disposable {
                 void kernel.dispose();
             }
             this.associatedDocuments.delete(event.notebook);
+            // Possible we have a live local kernel associated with this.
+            // this.kernelProvider.kernels.forEach((item) => item.connectedResourceUris.delete(event.notebook.uri.toString()));
```

Comment:
What's the connectedResourceURIs for? Looks like code you had for something that's not being used at the moment?

Reply:
Yes, I wanted to keep track of the number of connections associated with a local live kernel. So we can popular the numberOfConnections property in the Kernel Model, but felt that was somethign we could add later with a little bit more thought into that implementation.

E.g. assume we connect a notebook to the same live kernel used by interative window, then ideally it would be good to know that there are 2 connections associated with that kernel. However that's not necessarily part of this PR and I felt i could add that later.
Will remove that commented block of code.